site/_data/config/bk_server.yaml

```diff
@@ -43,8 +43,8 @@ groups:
     description: Interval to watch whether bookie is dead or not, in milliseconds.
     default: 1000
   - param: flushInterval
-    description: How long the interval to flush ledger index pages to disk, in milliseconds. Flushing index files will introduce much random disk I/O. If separating journal dir and ledger dirs each on different devices, flushing would not affect performance. But if putting journal dir and ledger dirs on same device, performance degrade significantly on too frequent flushing. You can consider increment flush interval to get better performance, but you need to pay more time on bookie server restart after failure.
-    default: 100
+    description: When entryLogPerLedgerEnabled is enabled, checkpoint doesn't happens when a new active entrylog is created / previous one is rolled over. Instead SyncThread checkpoints periodically with 'flushInterval' delay (in milliseconds) in between executions. Checkpoint flushes both ledger entryLogs and ledger index pages to disk.  Flushing entrylog and index files will introduce much random disk I/O. If separating journal dir and ledger dirs each on different devices, flushing would not affect performance. But if putting journal dir and ledger dirs on same device, performance degrade significantly on too frequent flushing. You can consider increment flush interval to get better performance, but you need to pay more time on bookie server restart after failure. This config is used only when entryLogPerLedgerEnabled is enabled.
+    default: 10000
   - param: allowStorageExpansion
```

Comment:
Arw we changing this?

Reply:
@eolivelli it was already 10000 - https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java#L338 I don't think the change touches it.

Reply:
there is discrepancy in the default value of this config. In ServerConfiguration.java it is mentioned as 10000 (10 secs) but in this file it is mentioned as 100. So I'm correcting it.